puma/hlplots/results.py

```diff
@@ -130,6 +147,8 @@ def add_taggers_from_file(  # pylint: disable=R0913
         # load data
         reader = H5Reader(file_path, precision="full")
         data = reader.load({key: var_list}, num_jets)[key]
+        # filter nan values
+        data = filter_nan(data)
```

Comment:
Thanks @dkobylianskii! I think we should guard this behind and `if` and add an argument `remove_nan` to `Results.__init__`, so that the default behaviour remains and an error. The user can then choose to override this.

Reply:
Thanks for the suggestion! I added this argument